### PR TITLE
added raw as a supported format for properties get

### PR DESCRIPTION
**Pull request by Wyvinar**

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: README.md [hunk 1]
```diff
@@ -322,7 +322,7 @@ A complete list of supported parameters for the `runs download` command is avail
 This command retrieves details of properties in a namespace.
 
 Properties in a namespace can be filtered out by using `--prefix`, `--infix` and/or `--suffix`, or `--name`.
-Currently one format is supported: "summary".
+Two formats are supported: 'summary', 'raw'
```
Comment (techcobweb) [inline README.md:325]: The default is summary.

Comment (techcobweb) [reply README.md:325]: Perhaps adding an example of the output with the raw format for two properties ?

File: pkg/properties/propertiesGet_test.go [hunk 2]
```diff
@@ -625,3 +625,30 @@ func TestInvalidPropertyNameReturnsEmpty(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, expectedOutput, console.ReadText())
 }
+
+func TestValidNamespaceFormatRawReturnsOk(t *testing.T) {
```
Comment (techcobweb) [inline pkg/properties/propertiesGet_test.go:629]: Nice unit test

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**